### PR TITLE
Create weekly-update.2015-04-24.md

### PR DESCRIPTION
**Pull request by Gioyik**
CC #64

### CODE REVIEW

File: weekly-updates/weekly-update.2015-04-24.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* NPM founder talks about io.js at [InfoWorld](http://www.infoworld.com/article/2910594/node-js/npm-founder-foresees-merger-node-js-io-js.html)
+* Proposal to add mikeal, mscdex, shigeki as new TC members. [iojs/io.js#1483](https://github.com/iojs/io.js/issues/1483#issuecomment-95128140)
+
+### Upcoming Events
+
+* [JSConf Uruguay](http://jsconf.uy) tickets are on sale, April 24th & 25th at Montevideo, Uruguay
+* [NodeConf Adventure](http://nodeconf.com/) tickets are on sale, June 11th - 14th at Walker Creek Ranch, CA
+* [CascadiaJS](http://2015.cascadiajs.com/) tickets are on sale, July 8th - 10th at Washington State
+* [NodeConf EU](http://nodeconf.eu/) tickets are on sale, September 6th - 9th at Waterford, Ireland
+* [nodeSchool Tokyo](http://nodejs.connpass.com/event/13182/) will be held in April 12th at Tokyo, Japan
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-04-24.md]: NodeSchool Tokyo is finished. :)

Comment (Gioyik) [inline weekly-updates/weekly-update.2015-04-24.md]: Sorry, I used a past one as template and didn't notice this.

**Comment by yosuke-furukawa** [comment]
LGTM. next release is v2.0!! so excited.

**Comment by julianduque** [comment]
LGTM - Publishing to Medium, will tweet tomorrow morning

**Comment by julianduque** [comment]
Tweeted just now, figure that is still afternoon in San Francisco, we can RT tomorrow too

https://twitter.com/official_iojs/status/591778312504135680

Great job as always @Gioyik

**Comment by diagramatics** [comment]
Notable changes has an extra empty bullet point on Medium, and that's it.

**Comment by julianduque** [comment]
@diagramatics fixed!

**Pull request merged**